src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -178,16 +171,12 @@ const SummaryBar = props => {
   };
 
   if (userProfile !== undefined && leaderData !== undefined) {
-    const infringements = getInfringements(userProfile);
-    const badges = getBadges(userProfile);
-    const { firstName, lastName, email, _id } = userProfile;
-    let totalEffort = parseFloat(leaderData.find(x => x.personId === asUser).tangibletime);
-    const weeklyCommittedHours = userProfile.weeklyCommittedHours
-      ? userProfile.weeklyCommittedHours
+    const weeklyCommittedHours = userProfile.weeklycommittedHours
```

Comment:
@RJBornToWin This block should stay like it is? There's a correct "weeklycommittedHours" and a misspelled "weeklyComittedHours" prop being used together. This was already on the code before your refactoring process btw.